com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/ShaderPass/ShaderPassRayTracingSubSurface.hlsl

```diff
@@ -4,6 +4,8 @@
 [shader("closesthit")]
 void ClosestSubSurface(inout RayIntersectionSubSurface rayIntersection : SV_RayPayload, AttributeData attributeData : SV_IntersectionAttributes)
 {
+    UNITY_XR_ASSIGN_VIEW_INDEX(DispatchRaysIndex().z);
```

Comment:
I don't understand why this is done

Reply:
As discussed over Slack, the code in BuildFragInputsFromIntersection() relies on _WorldSpaceCameraPos which is different for each eye in VR and will be indexed by the view index coming from DispatchRaysIndex().z.